media/magicavoxel_vox.ksy

```diff
@@ -0,0 +1,187 @@
+meta:
+  id: vox
+  file-extension: vox
+  application: MagicaVoxel
+  endian: le
+  license: MIT
+  title: MagicaVoxel File
+doc-ref: 'https://ephtracy.github.io/ MagicaVoxel Homepage'
+doc-ref: 'https://github.com/ephtracy/voxel-model/blob/master/MagicaVoxel-file-format-vox.txt Format Description'
```

Comment:
Unfortunately, it won't work like that. YAML in general (except when using a very special parser) disallows to have several instances of the same key, thus we'll get only one of these (the latter, if I recall it correctly). There has been a request to support multiple `doc-ref` entires somehow (https://github.com/kaitai-io/kaitai_struct/issues/269), but it's still not implemented.

Not a big deal, of course.